gcd/src/main/java/org/spine3/server/storage/datastore/DsAggregateStorage.java

```diff
@@ -232,22 +179,22 @@ public Key apply(@Nullable Entity input) {
 
         final Collection<Entity> filteredEntities = Collections2.filter(eventEntities,
                                                                         new IsActiveAggregateId(inactiveAggregateKeys));
-        final List<AggregateStorageRecord> immutableResult = Entities.entitiesToMessages(filteredEntities, TYPE_URL);
-        final List<AggregateStorageRecord> records = Lists.newArrayList(immutableResult);
+        final List<AggregateEventRecord> immutableResult = Entities.entitiesToMessages(filteredEntities, TYPE_URL);
```

Comment:
It is a generic method: `static <M extends Message> List<M> entitiesToMessages`
It may be used not only with the `AggregateEventRecords`.
Considering this, I think `entitiesToMessages` is a proper name for the method.